src/Baitapchuyentien/Account.java

```diff
@@ -0,0 +1,70 @@
+package Baitapchuyentien;
+
+import java.util.Scanner;
+
+public class Account {
+	private String id;
+	private String name;
+	private int balance = 0;
+
+	public Account(String id, String name, int balace) {
+		this.id = id;
+		this.name = name;
+		this.balance = balace;
+	}
+
+	public String getId() {
+		return id;
+	}
+
+	public void setId(String id) {
+		this.id = id;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public int getBalance() {
+		return balance;
+	}
+
+	public void setBalance(int balance) {
+		this.balance = balance;
+	}
+
+	public void credit(int amount) {
+		if (amount < 0) {
+			System.out.println("Số tiền nhập không chính xác");
+		} else {
+			this.balance = balance + amount;
+			System.out.println("Nạp tiền thành công,số tiền tài khoản là:" + balance);
+		}
```

Comment:
suggest nhỏ để kiệm code và nhìn pro hơn nè cu
```suggestion
               if (amount < 0) {
			System.out.println("Số tiền nhập không chính xác");
			return;
		}
		this.balance = balance + amount;
		System.out.println("Nạp tiền thành công,số tiền tài khoản là:" + balance);
```